.buildkite/serverless.beats.tests.yml

```diff
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  DOCKER_REGISTRY: "docker.elastic.co"
+  VAULT_PATH: "kv/ci-shared/observability-ingest/cloud/gcp"
+
+steps:
+  - label: "Serverless Beats Tests"
+    key: "serverless-beats-integration-tests"
+    retry:
+      automatic:
+        limit: 1
+    command: |
+      .buildkite/scripts/steps/beats_tests.sh
```

Comment:
do you need the artifacts to be available? I see a command about downloading them 

>       buildkite-agent artifact download "build/distributions/**" . $BUILDKITE_BUILD_ID

Reply:
It turns out, no :)